app/assets/javascripts/controllers.js

```diff
@@ -17,6 +17,27 @@ Prometheus.Graph = {
     's': 1
   },
 
+  durationToString: function(seconds) {
+    // TODO(stuart): this is always returning seconds
+    // look to round to a certain amount based on the units
```

Comment:
Ah sorry, I'll fix this. Basically, in broken no good engrish, it's saying "This function, for where it is being used, will always return a duration in seconds. Look at rounding to the nearest value." I don't think this is necessarily valid that we want to round, and it should be pretty self-evident that only perfectly divisible time units will be days/weeks/years, so I'm going to delete the comment.